boards/DJI_Board_TypeA/Middlewares/ST/STM32_USB_Device_Library/Class/CDC/Src/usbd_cdc.c

```diff
@@ -446,8 +446,6 @@ __ALIGN_BEGIN static uint8_t USBD_CDC_OtherSpeedCfgDesc[USB_CDC_CONFIG_DESC_SIZ]
   * @{
   */
 
-static USBD_CDC_HandleTypeDef hcdc_mem;
```

Comment:
Although this is an auto-generated file, but do not change this. This is a internal usb bug that fixed by our god a year ago.